woocommerce-services.php

```diff
@@ -9,7 +9,7 @@
  * Domain Path: /i18n/languages/
  * Version: 1.26.3
  * WC requires at least: 3.5.5
- * WC tested up to: 6.6.1
+ * WC tested up to: 6.7.0
```

Comment:
```suggestion
 * WC tested up to: 6.8
```

Reply:
I've updated the WC tested up to version in the plugin header and readme.txt